crates/forge-runner/src/forge_config.rs

```diff
@@ -90,3 +90,19 @@ impl<'a> RuntimeConfig<'a> {
         }
     }
 }
+
+#[derive(Debug, PartialEq)]
+pub struct SierraTestCodePathConfig {
+    pub package_name: String,
+    pub sierra_test_code_dir: Utf8PathBuf,
+}
+
+impl SierraTestCodePathConfig {
+    #[must_use]
+    pub fn path_for_sierra(&self, tests_location: &str) -> Utf8PathBuf {
```

Comment:
No clue why we need a path config and we build path from it.

Reply:
PathConfig is relevant for the whole package while the path is for a specific crate (the one from `src` or `tests`)

Reply:
Removed